freetextresponse/models.py

```diff
@@ -0,0 +1,242 @@
+"""
+Handle data access logic for the XBlock
+"""
+from __future__ import absolute_import
+
+from enum import Enum
+from django.db import IntegrityError
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I really hope I'm wrong about this one, I seriously do. [This issue have wasted a lot of my time for support requests, see this thread in Open edX Discuss](https://discuss.openedx.org/t/translate-the-default-xblock-data/213).

I don't think my dreams would come true immediately, so if you get exceptions regarding that issue, please use the `ugettext_noop` function instead.

```suggestion
from django.utils.translation import ugettext_noop as _
```